.github/workflows/dast.yml

```diff
@@ -40,18 +40,12 @@ jobs:
       - name: Check API Health
         run: ./health-check.sh
 
-      - name: Pull ZAP OWASP Image
-        run: docker pull owasp/zap2docker-weekly
-
-      - name: Create docker network
-        run: docker network create zapnet
```

Comment:
Just wondering why the net was taken out.

Reply:
Ah, yeah.  If the network was going to be used, the TI service also needed to run inside the network.  It wasn't.  So the network wasn't buying us anything.  Instead the TI service is being exposed to the host network which is why we do the entire `ip -f inet -o addr show docker0 | awk '{print $4}' | cut -d '/' -f 1` thing to get the host IP address.